drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/test/java/org/drools/workbench/screens/guided/dtable/client/widget/table/popovers/definitions/BRLActionColumnDefinitionBuilderTest.java

```diff
@@ -102,10 +100,5 @@ private void setupBRLActionColumn() {
         brl.getChildColumns().add(new BRLActionVariableColumn("f2",
                                                               DataType.TYPE_NUMERIC_INTEGER));
         model.getActionCols().add(brl);
-
-        when(dmo.getFieldType(eq("Person"),
-                              eq("name"))).thenReturn(DataType.TYPE_STRING);
-        when(dmo.getFieldType(eq("Person"),
-                              eq("age"))).thenReturn(DataType.TYPE_NUMERIC_INTEGER);
```

Comment:
@caponetto Ok, let' generalize this case as well. Which is the reason of all the tests logic you removed? Considering I don't see changes on code logic, I can suppose they were unused / useless for the test.